homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -198,10 +213,11 @@ class PropertyZWaveJSEntityDescription(
         entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
     ),
     NotificationZWaveJSEntityDescription(
-        # NotificationType 9: System - State Id's 1, 2, 6, 7
+        # NotificationType 9: System - State Id's 1, 2, 3, 4, 6, 7
         key=NOTIFICATION_SYSTEM,
-        states=("1", "2", "6", "7"),
+        states=("1", "2", "3", "4", "6", "7"),
```

Comment:
was you addition of state id's 3 and 4 intended ?

Reply:
Yes, these should also work as binary sensors even if the event contains some extra parameters with failure codes.